File: src/main/java/net/noncore/fdx/views/components/FileLine.java
package net.noncore.fdx.views.components;

import javafx.fxml.FXML;
import javafx.scene.control.Label;
import javafx.scene.layout.AnchorPane;
import net.noncore.fdx.views.View;

public class FileLine extends AnchorPane implements View {
    @FXML
    private Label name;
    @FXML
    private Label size;
    @FXML
    private Label date;

    public FileLine() {
        load("FileLine.fxml");
    }
}

File: src/main/kotlin/net/noncore/fdx/views/FileLine.kt
package net.noncore.fdx.views

import javafx.scene.control.Label
import javafx.scene.layout.AnchorPane
import tornadofx.*

class FileLine : View("File Line") {
    val name: Label by fxid()
    val size: Label by fxid()
    val date: Label by fxid()

    override val root: AnchorPane by fxml("/FileLine.fxml")
}
